portfolio/src/main/webapp/script.js

```diff
@@ -106,7 +106,283 @@ function addCommentToDom(element, comment){
 }
 /** Creates a map and adds it to the page. */
 function createMap() {
-  const map = new google.maps.Map(
+    var r1 = {lat: 34.082119, lng: -118.378906};
+    var r2 = {lat: 34.0450563, lng: -118.2539964};
+    var r3 = {lat: 34.0210869, lng: -118.4019363};
+    var r4 = {lat: 34.095017, lng: -118.400399};
+    var r5 = {lat: 33.407696, lng: -118.3739947};
+
+    const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
-}
\ No newline at end of file
+      {center: {lat: 34.052240, lng: -118.243340}, zoom: 11});
+    var marker1 = new google.maps.Marker({position: r1, map: map});
+    var marker1 = new google.maps.Marker({position: r2, map: map});
+    var marker2 = new google.maps.Marker({position: r3, map: map});
+    var marker3 = new google.maps.Marker({position: r4, map: map});
+    var marker4 = new google.maps.Marker({position: r5, map: map});
+}
+/*
+<script src="https://maps.googleapis.com/maps/api/staticmap?key=AIzaSyAF6Z09E_v1B5Jm0N6JkTRI-31KYqfxHwI&center=47.65,-122.35&zoom=12&format=png&maptype=roadmap&style=element:geometry%7Ccolor:0xebe3cd&style=element:geometry.fill%7Ccolor:0xffe29a%7Csaturation:-10%7Clightness:10%7Cweight:4.5&style=element:geometry.stroke%7Cweight:2&style=element:labels.text%7Cweight:2.5&style=element:labels.text.fill%7Ccolor:0x523735%7Cweight:2&style=element:labels.text.stroke%7Ccolor:0xf5f1e6%7Cweight:3&style=feature:administrative%7Celement:geometry.stroke%7Ccolor:0xc9b2a6&style=feature:administrative.land_parcel%7Celement:geometry.stroke%7Ccolor:0xdcd2be&style=feature:administrative.land_parcel%7Celement:labels.text.fill%7Ccolor:0xae9e90&style=feature:landscape.natural%7Celement:geometry%7Ccolor:0xdfd2ae&style=feature:poi%7Celement:geometry%7Ccolor:0xdfd2ae&style=feature:poi%7Celement:labels.text.fill%7Ccolor:0x93817c&style=feature:poi.park%7Celement:geometry.fill%7Ccolor:0xa5b076&style=feature:poi.park%7Celement:labels.text.fill%7Ccolor:0x447530&style=feature:road%7Celement:geometry%7Ccolor:0xf5f1e6&style=feature:road.arterial%7Celement:geometry%7Ccolor:0xfdfcf8&style=feature:road.highway%7Celement:geometry%7Ccolor:0xf8c967&style=feature:road.highway%7Celement:geometry.stroke%7Ccolor:0xe9bc62&style=feature:road.highway.controlled_access%7Celement:geometry%7Ccolor:0xe98d58&style=feature:road.highway.controlled_access%7Celement:geometry.stroke%7Ccolor:0xdb8555&style=feature:road.local%7Celement:labels.text.fill%7Ccolor:0x806b63&style=feature:transit.line%7Celement:geometry%7Ccolor:0xdfd2ae&style=feature:transit.line%7Celement:labels.text.fill%7Ccolor:0x8f7d77&style=feature:transit.line%7Celement:labels.text.stroke%7Ccolor:0xebe3cd&style=feature:transit.station%7Celement:geometry%7Ccolor:0xdfd2ae&style=feature:water%7Celement:geometry.fill%7Ccolor:0xb9d3c2&style=feature:water%7Celement:labels.text.fill%7Ccolor:0x92998d&size=480x360
+"></script>
+function createMap() {
```

Comment:
Let's avoid submitting big chunks of commented code. If you want to use this PR to try out the styled map that is fine with me so you can continue working on this rather than commenting it out here :)